app/views/relation/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<% unless @relations.empty? %>
+  <div class="panel panel-default show-tools">
+    <div class="panel-heading">
+      <%= t('geoblacklight.relations.title') %>
+    </div>
+    <div class="panel-body">
+      <ul class="nav">
+        <% unless (@relations.ancestors['numFound'].to_i == 0) %>
+          <b><%= t('geoblacklight.relations.ancestor') %></b>
+          <% @relations.ancestors['docs'].each do |ancestor| %>
+            <li>
+              <%= link_to solr_document_path(ancestor['layer_slug_s']) do %>
+                <span class='geoblacklight geoblacklight-relations-ancestor'></span> <%= ancestor['dc_title_s'] %>
+              <% end %>
+            </li>
+          <% end %>
+        <% end %>
+        <% unless (@relations.descendants['numFound'].to_i == 0) %>
+          <b><%= t('geoblacklight.relations.descendant') %>(<%= @relations.descendants['numFound'] %>)</b>
```

Comment:
Could you add a space between the descendant string and the open paren? I think there was one before, maybe just obscured by the fact that this was previously spanning two lines.